sdk/confidentialledger/azure-security-confidentialledger/swagger/README.md

```diff
@@ -33,7 +35,7 @@ data-plane: true
 credential-types: tokencredential
 credential-scopes: https://confidential-ledger.azure.com/.default
 title: ConfidentialLedgerIdentityClient
-use: '@autorest/java@4.0.62'
+use: '@autorest/java@4.1.0'
```

Comment:
Let's use the namespace `com.azure.security.confidentialledger.certificate` on line number 31 for this client.